### PR TITLE
DigitalIdentity should have been DigitalCredential

### PR DESCRIPTION
**Pull request by marcoscaceres**
Also align and defer to Verifiable Credential defined terms.

***
<a href="https://pr-preview.s3.amazonaws.com/WICG/digital-identities/pull/79.html" title="Last updated on Feb 16, 2024, 8:54 PM UTC (3d8ca18)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/digital-identities/79/26ee01b...3d8ca18.html" title="Last updated on Feb 16, 2024, 8:54 PM UTC (3d8ca18)">Diff</a>

### CODE REVIEW

**Review by samuelgoto — APPROVED** [review]
LGTM

**Review by OR13 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -43,14 +43,14 @@
     <section id="abstract">
       <p>
         This document specifies an API to allow user agents to mediate access
-        to, and representing, a verifiably-issued digital identity (e.g. a
+        to, and representing, a verifiably-issued digital credential (e.g. a
```
Comment (TallTed) [inline index.html:46]:
```suggestion
        to, and representing, a verifiably-issued digital credential (e.g., a
```

File: index.html [hunk 2]
```diff
@@ -169,16 +139,31 @@ <h2>
...
     partial dictionary CredentialRequestOptions {
+      DigitalCredentialRequestOptions digital;
+    };
+    </pre>
+    <h3>
+      The `digital` member
+    </h3>
+    <p>
+      The <dfn data-dfn-for="CredentialRequestOptions">digital</dfn> member
+      allow for options to configure the request for a [=digital credential=].
```
Comment (TallTed) [inline index.html:150]:
```suggestion
      allows for options to configure the request for a [=digital credential=].
```

**Comment by marcoscaceres** [comment]
@TallTed, fixed the typos in a followup! Thanks for pointing those out.

**Pull request merged**